### ISSUE TITLE
Automatic Number Plate Recognition question

### ISSUE THREAD

**Issue by dsEngineer** [ispysoftware/iSpy#127]
Does anyone know if the License plate recognition plugin have the ability to perform detection on a moving vehicle?

Which brings up another question. Where do people normally post questions about iSpy?

Thanks!

**Comment by ispysoftware** [comment]
Yes it does. A moving vehicle is just a series of images. As long as the plate is legible it should work. There is a reddit forum linked on the website main menu  under more
